Sources/SwiftSentry/SentrySDK.swift

```diff
@@ -72,6 +72,17 @@ public enum SentrySDK {
             }, nil)
         }
 
+        #if os(Windows)
+        sentry_options_set_on_crash(o, { uctx, event, _ -> sentry_value_t in
+            guard let fatalErrorMessageBuf = getFatalErrorMessageBuffer() else {
```

Comment:
Can we do this super early on init and cache the location? That should ensure that we execute almost no code in the failed environment.

Reply:
Done